packages/office-ui-fabric-react/src/components/pickers/Suggestions/SuggestionsController.ts

```diff
@@ -12,12 +12,14 @@ export class SuggestionsController<T> {
     this.currentIndex = -1;
   }
 
-  public updateSuggestions(newSuggestions: T[]) {
+  public updateSuggestions(newSuggestions: T[], setSelected: boolean) {
```

Comment:
instead of setSelected being a boolean, why not make it selectedIndex? That way you can pass in any index and make sure you get the right one selected.